### DISCUSSION TITLE
Failing to run user_creation collection

### DISCUSSION THREAD

**Discussion by TechServ2022** [egovernments/Digit-Core#45] [Local Governance]
1. DIGIT Vesrion - 2.6
2. Attempting Full Installation.
3. Installation with AWS.
4. Documentation link - https://docs.digit.org/v/v2.6/setup/install-on-cloud/on-aws/7.-bootstrap-digit
5. Step Number 7, section 2.
6. Ubuntu version 18.04.
7.  EC2 - m4x.large 7 instances

**Issue:**
While running the user_creation collection in postaman, when running the accessToken_genration API.
It brings the Error: getaddrinfo ENOTFOUND quickstart.local.digit
![image](https://user-images.githubusercontent.com/101874385/179502496-23c5075f-ebff-4cd8-adbf-97d449d272d7.png)

Then in the body section, I changed the value from quickstart.local.digit to my.stcldigit.com (my domain)
It brings a JsonError as stated JSONError: Unexpected token '<' at 1:1<html>^

![image](https://user-images.githubusercontent.com/101874385/179502885-d5d83ef5-df52-43de-bed6-63d448ed2d59.png)

**Comment by shashwat-egov** [comment]
The following cURL is working for all others who have been through this tutorial, please cross check the request body that you are using with this cURL - 
`curl --location --request POST 'http://localhost:8080/user/oauth/token' \
--header 'authority: quickstart.local.digit' \
--header 'sec-ch-ua: " Not A;Brand";v="99", "Chromium";v="90", "Google Chrome";v="90"' \
--header 'accept: application/json, text/plain, */*' \
--header 'authorization: Basic ZWdvdi11c2VyLWNsaWVudDo=' \
--header 'sec-ch-ua-mobile: ?0' \
--header 'user-agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.212 Safari/537.36' \
--header 'content-type: application/x-www-form-urlencoded' \
--header 'origin: http://quickstart.local.digit' \
--header 'sec-fetch-site: same-origin' \
--header 'sec-fetch-mode: cors' \
--header 'sec-fetch-dest: empty' \
--header 'referer: http://quickstart.local.digit/employee/user/login' \
--header 'accept-language: en-GB,en;q=0.9' \
--data-urlencode 'username=GRO' \
--data-urlencode 'password=eGov@4321' \
--data-urlencode 'grant_type=password' \
--data-urlencode 'scope=read' \
--data-urlencode 'tenantId=pg.citya' \
--data-urlencode 'userType=EMPLOYEE'`

**Reply by TechServ2022** [reply]
This is the reply I am getting

{"timestamp":"25-07-2022 18:30:23","status":401,"error":"Unauthorized","message":"Full authentication is required to access this resource","path":"/user/oauth/token"}curl: (3) Host name ' --header' contains bad letter
curl: (3) Port number ended with ' '
curl: (3) Host name ' --header' contains bad letter
curl: (3) Port number ended with ' '
curl: (3) Host name ' --header' contains bad letter
curl: (3) Port number ended with ' '
curl: (3) Host name ' --header' contains bad letter
curl: (3) Port number ended with ' '
curl: (3) Host name ' --header' contains bad letter
curl: (3) Port number ended with ' '
curl: (3) Host name ' --header' contains bad letter
curl: (3) Port number ended with ' '
curl: (3) Host name ' --header' contains bad letter
curl: (3) Port number ended with ' '
curl: (3) Host name ' --header' contains bad letter
curl: (3) Port number ended with ' '
curl: (3) Host name ' --header' contains bad letter
curl: (3) Port number ended with ' '
curl: (3) Host name ' --header' contains bad letter
curl: (3) Port number ended with ' '
curl: (3) Host name ' --header' contains bad letter
curl: (3) Port number ended with ' '
curl: (3) Host name ' --header' contains bad letter
curl: (3) Port number ended with ' '
curl: (3) Host name ' --header' contains bad letter
curl: (3) Port number ended with ' '
curl: (3) Host name ' --data-urlencode' contains bad letter
curl: (6) Could not resolve host: username=GRO
curl: (3) Host name ' --data-urlencode' contains bad letter
curl: (7) Host name ' --data-urlencode' contains bad letter
curl: (3) Host name ' --data-urlencode' contains bad letter
curl: (6) Could not resolve host: grant_type=password
curl: (3) Host name ' --data-urlencode' contains bad letter
curl: (6) Could not resolve host: scope=read
curl: (3) Host name ' --data-urlencode' contains bad letter
curl: (6) Could not resolve host: tenantId=pg.citya
curl: (3) Host name ' --data-urlencode' contains bad letter
curl: (6) Could not resolve host: userType=EMPLOYEE

**Reply by talele08** [reply]
There seems to be some problem when copying cURL from git discussion board. @shashwat-egov has pushed the same cURL on this link [cURL](https://raw.githubusercontent.com/egovernments/DIGIT-OSS/bbc6aa4a095c8718dbd0790e41cfe4a3534c5a0c/user_auth_curl.txt). Please copy from here.(Modify the host in the cURL as per the setup)

**Reply by eGovMan** [reply]
@TechServ2022  Is this issue resolved?
